src/service/project/tempSaveProject.ts

```
@@ -0,0 +1,72 @@
+import { prisma } from "../../config/prisma";
+import { Response } from "express";
+import { AuthenticatedRequest, BasicResponse } from "../../types";
+import { RegisterProjectBody } from "../../types/project";
+import { ProjectState } from "../../config/prisma";
+
+// 임시 저장
+
+export const tempSaveProject = async (
+  req: AuthenticatedRequest<{}, {}, RegisterProjectBody>,
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)